### PR TITLE
Release 5.6.1

### PR DESCRIPTION
**Pull request by weeeBox**
#### Improvements

* Accessibility improvements.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 2021-01-04 - v5.6.1
+
+#### Improvements
+
+* Accessibility improvements.
```
Comment (MatthewCallis) [inline CHANGELOG.md:5]: Could you note the actual accessibility changes (looks like Range labels and the `setContentDescription` addition) and note the change of `ApptentiveNestedScrollView` to `NestedScrollViewHelper` here too.

**Review by MatthewCallis — CHANGES_REQUESTED** [review]
Looks good but lets be more explicit in the change log for easier bisecting (when needed) as we have new folks coming in.

File: CHANGELOG.md (hunk 1 above)
Comment (weeeBox) [reply CHANGELOG.md:5]: The transition from `ApptentiveNestedScrollView` to `NestedScrollViewHelper ` is more like a "private" API change and might not bring many benefits to the "public" changelog.

Comment (weeeBox) [reply CHANGELOG.md:5]: I vote we merge it as-is and be more pro-active with the changelog updates in the future.

Comment (MatthewCallis) [reply CHANGELOG.md:5]: Is there some reason you can't start making those additions now?

**Review by MatthewCallis — APPROVED** [review]

**Pull request merged**